src/page/auth/account/user-account.tsx

```diff
@@ -57,51 +120,100 @@ const UserAccount = () => {
-                variant='outlined'
-                label='Full Name'
-                value={fullName}
-                style={{ marginTop: '1rem' }}
-                fullWidth
-                InputProps={{
-                  sx: {
-                    '& .MuiOutlinedInput-notchedOutline': {
-                      border: 'none'
-                    },
-                    fontWeight: 'bold'
-                  }
-                }}
-              />
-              <TextField
-                variant='outlined'
-                label='Location'
-                value={account?.data?.location}
-                style={{ marginTop: '1rem' }}
-                fullWidth
-                InputProps={{
-                  sx: {
-                    '& .MuiOutlinedInput-notchedOutline': {
-                      border: 'none'
-                    },
-                    fontWeight: 'bold'
-                  }
-                }}
-              />
-              <TextField
-                variant='outlined'
-                label='Email'
-                value={account?.data?.email}
-                style={{ marginTop: '1rem' }}
-                fullWidth
-                InputProps={{
-                  sx: {
-                    '& .MuiOutlinedInput-notchedOutline': {
-                      border: 'none'
-                    },
-                    fontWeight: 'bold'
-                  }
-                }}
-              />
+
+              <FormProvider methods={methods} onSubmit={handleSubmit(onSubmit)}>
+                <FormField
+                  name={KEY.EMAIL}
+                  submitting={isSubmitting}
+                  sent={isSubmitSuccessful}
+                  errors={errors}
+                  label='Email'
+                  placeholder={account?.data?.email}
+                  autoComplete='email'
+                  type='email'
+                  disabled={!isEditState}
+                  required
+                  isAccount
+                />
+                <FormField
+                  name={KEY.LOCATION}
+                  submitting={isSubmitting}
+                  sent={isSubmitSuccessful}
+                  errors={errors}
+                  label='Location'
+                  placeholder={account?.data?.location}
+                  autoComplete='location'
+                  disabled={!isEditState}
+                  required
+                  isAccount
+                />
+                {isEditState ? (
+                  <Fragment>
+                    <FormField
+                      name={KEY.FIRST_NAME}
+                      submitting={isSubmitting}
+                      sent={isSubmitSuccessful}
+                      errors={errors}
+                      label='First name'
+                      placeholder={account?.data?.firstname}
+                      autoComplete='firstname'
+                      disabled={!isEditState}
+                      required
+                      isAccount
```

Comment:
Refactor this: add to backlog.

Reply:
ISSUE: https://github.com/ballyalley-o/tccp/issues/155